README.md

```diff
@@ -94,6 +94,10 @@ implementation 'io.grpc:grpc-stub:1.54.1'
 compileOnly 'org.apache.tomcat:annotations-api:6.0.53' // necessary for Java 9+
 ```
 
+For [Bazel](https://bazel.build), you can either
+[use Maven](https://github.com/bazelbuild/rules_jvm_external/blob/master/docs/bzlmod.md)
```

Comment:
Given java_grpc_library isn't available, I don't think linking to bzlmod helps anybody.

Reply:
ACK, let me remove that...

Reply:
@ejona86 done. Squash ?